src/NServiceBus.Core/Pipeline/BehaviorContext.cs

```diff
@@ -19,6 +20,11 @@ public IBuilder Builder
             }
         }
 
+        public Task RaiseNotification<T>(T notification) where T : INotification
+        {
+            return Builder.Build<EventAggregator>().Raise(notification);
```

Comment:
For reference we do the same with the pipeline cache

https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/Transports/TransportReceiver.cs#L64